pipelines/models/tensorflow-housing/README.md

```diff
@@ -0,0 +1,24 @@
+
+# How was this model generated?
+
+## Data source
+
+The model uses
+https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
+see
+https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
+for more information about the data.
+
+## Generating model
+
+With https://github.com/mlflow/mlflow checked out and specifically
+the training script
+https://github.com/mlflow/mlflow/blob/master/examples/tensorflow/train.py
+present in that repo, command
+```
+mlflow run mlflow/examples/tensorflow
+```
+was used to generate the model. The result was stored in
+a `mlruns/0/<run-id>/artifacts/model/` directory, and copied over to
```

Comment:
```suggestion
present in that repo, The following command was used to generate the model:
\```
mlflow run mlflow/examples/tensorflow
\```
The result was stored in an `mlruns/0/<run-id>/artifacts/model/` directory,
 and copied over to
```

I don't know how to properly escape a code block in a suggestion, so don't "accept" this suggestion as-is.

The intention is just to suggest rephrasing the sentence so that the text is all before the code block rather than before and after.